src/Dictionaries/IDictionary.h

```diff
@@ -167,4 +194,33 @@ inline void checkAttributeType(const IDictionaryBase * dictionary, const std::st
                         attribute_name, toString(attribute_type), toString(to)};
 }
 
+template <typename T>
+static const PaddedPODArray<T> &
+getColumnDataAsPaddedPODArray(const IDictionaryBase * dictionary, const ColumnPtr column, PaddedPODArray<T> & backup_storage)
```

Comment:
I don't like additional argument, which may not be used in some cases... It will add complexity to this code. Maybe better to pass result as the last argument and make this function return type void?

Reply:
Documented this method and moved to DictionaryHelpers.h.